docs/dxp/7.2.x/en/platform/app-builder/app-builder.md

```diff
@@ -0,0 +1,54 @@
+# Low-Code Application Development with App Builder
+
+Writing applications for Liferay DXP has never been easier. Introduced with Liferay DXP 7.3, App Builder provides a low-code application development solution. There are four concepts at the root of App Builder apps:
+
+- [Custom Objects](#custom-objects)
+
+- [Form Views](#form-views)
+
+- [Table Views](#table-views)
+
+- [Deployment](#deployment)
+
+These concepts are explained below. To skip straight to building an application, see the next article on [Creating Applications with App Builder](./creating-applications-with-app-builder.md).
+
+## Custom Objects
+
+Custom objects are the foundation of your App Builder applications. The custom object is like the model in a [Service Builder application](placeholder). The custom object defines what data fields can be contained in the application. For example, a Guestbook object would contain a _Name_ field and a _Message_ field, at least. To create applications based on a custom object, you have three tasks: create at least one form view, create at least one table view, and deploy the application.
```

Comment:
The text here gives the idea that Custom Objects are the only foundation of App Builder, when we will also have Native Objects in a near future. I would change this to say that "Objects are the foundation of you App Builder applications" and then, talk about custom objects as a branch of the Object concept. And later, we can update this doc to talk about the other branch, the Native Objects. What do you think? makes sense?